### PR TITLE
set locators for all objects in XMIR

### PR DESCRIPTION
**Pull request by yegor256**
Per #1460 

I added an extra attribute to all XMIR objects. The attribute contains unique locator of an object.

### CODE REVIEW

File: eo-parser/src/main/resources/org/eolang/parser/set-locators.xsl [hunk 1]
```diff
@@ -0,0 +1,95 @@
...
+              <xsl:text>Φ</xsl:text>
+            </xsl:when>
+            <xsl:otherwise>
+              <xsl:value-of select="$o/@name"/>
+            </xsl:otherwise>
+          </xsl:choose>
+        </xsl:when>
+        <xsl:otherwise>
+          <xsl:choose>
+            <xsl:when test="name($o) = 'o'">
```
Comment (volodya-lombrozo) [inline eo-parser/src/main/resources/org/eolang/parser/set-locators.xsl:71]: @yegor256 Could it be something except object `<o>` here?

**Review by volodya-lombrozo — CHANGES_REQUESTED** [review]
@yegor256 I believe we need more test cases here.

**Comment by yegor256** [comment]
@volodya-lombrozo very good catch! I added more tests and extra validation, please check again.

File: eo-parser/src/main/resources/org/eolang/parser/set-locators.xsl [hunk 2]
```diff
@@ -0,0 +1,103 @@
...
+                  <xsl:text>ρ</xsl:text>
+                </xsl:when>
+                <xsl:otherwise>
+                  <xsl:text>α</xsl:text>
+                  <xsl:value-of select="count($o/preceding-sibling::o)"/>
+                </xsl:otherwise>
+              </xsl:choose>
+            </xsl:when>
+            <xsl:when test="name($o) = 'objects'">
+              <xsl:text>ν0</xsl:text>
```
Comment (volodya-lombrozo) [inline eo-parser/src/main/resources/org/eolang/parser/set-locators.xsl:74]: @yegor256 Please, provide an example, where we can receive the `ν0`

**Review by volodya-lombrozo — CHANGES_REQUESTED** [review]
@yegor256 only one question about `ν0`. You can also squash commits to make the history more clear.

**Comment by yegor256** [comment]
@volodya-lombrozo you are right again, thanks! I removed the redundant piece of code, check 869ed32d

**Comment by yegor256** [comment]
@rultor merge

**Comment by yegor256** [comment]
@volodya-lombrozo `Φ.org.abc.tt.α2.φ` is the locator or `hello`. I will explain you this in person, it's a long story :)

**Pull request merged**